src/cozmo/robot.py

```diff
@@ -750,7 +765,9 @@ def _recv_msg_robot_state(self, evt, *, msg):
         self.left_wheel_speed  = util.speed_mmps(msg.leftWheelSpeed_mmps)
         self.right_wheel_speed = util.speed_mmps(msg.rightWheelSpeed_mmps)
         self.lift_height       = util.distance_mm(msg.liftHeight_mm)
-        self.battery_voltage           = msg.batteryVoltage
+        self.battery_voltage = msg.batteryVoltage
+        self.accelerometer = util.Vector3(msg.accel.x, msg.accel.y, msg.accel.z)
+        self.gyro = util.Vector3(msg.gyro.x, msg.gyro.y, msg.gyro.z)
```

Comment:
we should either align all these `=` symbols or none of them (either is fine with me)